### PR TITLE
fix(webpack): ignore child-compilers in RenderPlugin

### PR DESCRIPTION
**Pull request by herschel666**
This should™ resolve xing/hops/issues/761.

### CODE REVIEW

**Comment by robin-drexler** [comment]
Can you elaborate what hops-postcss does to run into this particular issue?

**Comment by herschel666** [comment]
It adds a child compiler `'mini-css-extract-plugin'` to the compilation, so the `RenderPlugin` hooks into the `additionalAssets` twice.

File: packages/webpack/lib/plugins/render.js [hunk 1]
```diff
@@ -20,6 +20,9 @@ exports.RenderPlugin = class RenderPlugin {
         )
       );
       compiler.hooks.compilation.tap('RenderPlugin', (compilation) => {
+        if (compilation.compiler.isChild()) {
```
Comment (dmbch) [inline packages/webpack/lib/plugins/render.js]: I did not know `Compiler` instances had an `isChild()` method - nice find!

Could we not also use it in the [`StatsPlugin`](https://github.com/untool/untool/blob/master/packages/webpack/lib/plugins/stats.js#L55)?

//cc @ZauberNerd

**Comment by robin-drexler** [comment]
Can we merge and release this, together with #235 to prepare a new hops release?
We'll add a bunch of tests in hops to ensure stuff has actually been fixed there too. :)

**Comment by dmbch** [comment]
This one is good to go, but I would love to have #235 investigated/understood a bit better to make sure we do not introduce a subtle regression somewhere.

**Comment by herschel666** [comment]
failed

@dmbch Can you merge this? It doesn't work via Github UI for me & in the Terminal I am causing a merge commit, what doesn't seem to be appreciated (referring to the git history of the project…).

**Comment by robin-drexler** [comment]
Merged on my machine™ @herschel666

**Pull request merged**